tekton/resources.yaml

```diff
@@ -51,5 +51,15 @@ metadata:
 spec:
   type: image
   params:
-   - name: url
-     value: cmd/eventlistenersink # Registry is provided via parameter, this is a hack see #569
\ No newline at end of file
+  - name: url
+    value: cmd/eventlistenersink  # Registry is provided via parameter, this is a hack see #569
+# ---
```

Comment:
Was this meant to be commented out?

Reply:
Yeah, the URL of the sink there is cluster specific so I didn't want to include a resource that would actually be applied with a broken url